.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
   - export PATH=$GOROOT/bin:$PATH
   - go version
   - go env
+  - go get -u github.com/matoous/godox
```

Comment:
`godox` may be added to `golangci-lint`

If not I might just extend our current `panic` finder to look for `TODO`, `FIXME` and `BUG`

I think this is the right decision. Currently people contribute code, but don't drive the tech debt down. This is also a good way to get things in the bug tracker so others can be aware/help out!